client/components/gsuite/gsuite-new-user-list/new-user.tsx

```diff
@@ -152,7 +152,7 @@ const GSuiteNewUser: FunctionComponent< Props > = ( {
 							onClick={ onUserRemove }
 						>
 							<Gridicon icon="trash" />
-							<span>{ translate( 'Remove User' ) }</span>
+							<span>{ translate( 'Remove user' ) }</span>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.